### ISSUE TITLE
Latex: Does not properly compile when PROJECT_BRIEF contains commas

### ISSUE THREAD

**Issue by msarehn** [doxygen/doxygen#9124]
**Describe the bug**
When the `PROJECT_BRIEF` contains commas, the `pdflatex` fails to produce a PDF.

**Expected behavior**
The generated TeX file should be compiled with `pdflatex` without issues.

**TeX output**
The generated TeX file contains this code (around line 130 in our case):
```tex
   \hypersetup{%
     colorlinks=true,%
     linkcolor=blue,%
     citecolor=blue,%
     unicode,%
     pdftitle=application,%
     pdfsubject=some description, containing a comma%
   }
```

The comma in the value for `pdfsubject` breaks the key-value parser used by that TeX package.

**To Reproduce**

You only need a very minimal Doxyfile:

```ini
# Doxyfile
DOXYFILE_ENCODING      = UTF-8
PROJECT_NAME           = "application"
PROJECT_NUMBER         =
PROJECT_BRIEF          = "some description, containing a comma"
```

Run `doxygen && cd latex && make` and it'll break with
```
! Package kvsetkeys Error: Undefined key `containing a comma'.

See the kvsetkeys package documentation for explanation.
Type  H <return>  for immediate help.
 ...

l.136     }
```

**Version**

Using the binary as downloaded from the website:
```
# doxygen --version
1.9.3 (c0b9eafbfb53286ce31e75e2b6c976ee4d345473)
```

```
# cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.3 LTS"
```

64 bit, fwiw.

**Stack trace**
Not applicable.

**Additional context**
Doxygen 1.8.17 from the Ubuntu 20.04 repo works fine. The TeX sources apparently don't contain the `PROJECT_BRIEF` in this case.

**Comment by albert-github** [comment]
The `ptdtitle` and `pdfsubject` in the properties page are an enhancement added with the 1.9.3 version.

I've just pushed a proposed patch, pull request #9125

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.4.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
